frontend/packages/shared/src/ext-urls.ts

```diff
@@ -1,5 +1,6 @@
 // Docs
-export const altinnDocsUrl = (relativeUrl: string) => `https://docs.altinn.studio/${relativeUrl}`;
+export const altinnDocsUrl = (relativeUrl: string, language: 'nb' | 'en' = 'nb') =>
+  `https://docs.altinn.studio/${language}/${relativeUrl}`;
```

Comment:
Yep, good catch! The plan is to use the same structure for all languages in docs, but it's not done yet. Will add a comment that this will need to be updated once the docs url structure is changed!